tasks/README.md

```diff
@@ -17,14 +17,25 @@ The Puppet Task ecosystem allows users to execute actions on target systems. A *
 
 ## Puppet Task Spec Versioning
 
-The task spec has a version and a revision. Changes that would break existing tasks will increment the version of the task spec. The revision of the task spec functions as a minor version and will be incremented for new features. The spec will document which revision a feature was added in.
-
-Orchestrator shipped in PE 2017.3 supports version 1 rev 1. [Bolt] supports version 1 rev 2.
+The task spec has a version and a revision. Changes that break existing tasks increment the version of the task spec. Revisions function as a minor version and are incremented for new features. This spec indicates in parentheticals the revision number for new features, for example "Running remote tasks (rev 4)." 
 
 Older versions of the task runner may not support tasks that rely on newer features of the task spec. The task spec version is not intended to capture this now. Authors should use the `puppet_task_version` field in the module's metadata to document such incompatibilities for users.
 
 Task Runners should document which version and revision of the task spec they support.
 
+### Revision history
+
+These are the revisions to this version of the task spec. 
+
+**Note**: Bolt always uses the latest version and revision of the task spec. Puppet Enterprise versions use a specific version and revision of the task spec, indicated below. 
+
+| Revision | Changes                  | PE version                |
+|----------|--------------------------|---------------------------|
+| 1        |                          | 2017.3.z 2018.1.z         |
+| 2        | Cross-platform tasks.    | 2019.0.0                  |
+| 3        | Multiple files per task. | 2019.0.1                  |
+| 4        | Hide private tasks.      |                           |
```

Comment:
Is rev 4 in 2019.0.2?

Reply:
No. I don't know when we'll add the "hide private tasks" feature. It's a bit different from the other ones since it doesn't prevent tasks any tasks from working.